src/components/ReportActionItem/ReportPreview.tsx

```diff
@@ -467,13 +469,12 @@ function ReportPreview({
                     accessibilityLabel={translate('iou.viewDetails')}
                 >
                     <View style={[styles.reportPreviewBox, isHovered || isScanning || isWhisper ? styles.reportPreviewBoxHoverBorder : undefined]}>
-                        {hasReceipts && (
-                            <ReportActionItemImages
-                                images={lastThreeReceipts}
-                                total={transactionsWithReceipts.length}
-                                size={CONST.RECEIPT.MAX_REPORT_PREVIEW_RECEIPTS}
-                            />
-                        )}
+                        <ReportActionItemImages
+                            images={lastThreeReceipts}
+                            total={allTransactions.length}
+                            size={CONST.RECEIPT.MAX_REPORT_PREVIEW_RECEIPTS}
+                            onPress={openReportFromPreview}
```

Comment:
Coming from checklist #53697, we don’t need to handle this action because the parent already handles it at https://github.com/abzokhattab/App/blob/ce344b5eef2ade348a24a9484c86e199e6ab8155/src/components/ReportActionItem/ReportPreview.tsx#L492.